src/index.html

```diff
@@ -199,20 +199,34 @@ <h2>最新バージョン</h2>
 									<span id="packages-check-date"></span>）
 								</div>
 							</button>
-							<button
-								type="button"
-								class="btn btn-primary mb-2"
-								id="install-package"
-							>
-								インストール
-							</button>
-							<button
-								type="button"
-								class="btn btn-primary mb-2"
-								id="install-script"
-							>
-								リストにないスクリプトのインストール
-							</button>
+							<div class="btn-group">
+								<button
+									type="button"
+									class="btn btn-primary mb-2"
+									id="install-package"
+								>
+									インストール
+								</button>
+								<button
+									type="button"
+									class="
+										btn btn-primary
+										mb-2
+										dropdown-toggle dropdown-toggle-split
+									"
+									data-bs-toggle="dropdown"
+									aria-expanded="false"
+								>
+									<span class="visually-hidden">Toggle Dropdown</span>
+								</button>
+								<ul class="dropdown-menu">
```

Comment:
How about this?
It makes the menu to right align.
https://getbootstrap.com/docs/5.1/components/dropdowns/#menu-alignment
```suggestion
								<ul class="dropdown-menu dropdown-menu-end">
```